avro-flink-serde/pom.xml

```diff
@@ -66,13 +66,13 @@
         <dependency>
             <groupId>software.amazon.glue</groupId>
             <artifactId>schema-registry-serde</artifactId>
-            <version>1.0.2</version>
+            <version>1.1.0</version>
         </dependency>
 
         <dependency>
             <groupId>software.amazon.glue</groupId>
             <artifactId>schema-registry-serde</artifactId>
-            <version>1.0.2</version>
+            <version>1.0.1</version>
```

Comment:
Is this correct?

Reply:
fixing thanks, test jars are not imported from local cache and hence maven has to use one on central repo. Using an older version.